common/tests/ads_api_types_test.ts

```diff
@@ -30,70 +31,38 @@ type AssertEqual<TestType, End> = End extends TestType
     : false
   : false;
 
-describe('Ads API Types', () => {
-  const CHILD_QUERY1 = buildQuery({
-    queryParams: ['b'],
-    queryFrom: 'test',
-  });
-  const CHILD_QUERY2 = buildQuery({
-    queryParams: ['c'],
-    queryFrom: 'test',
-  });
-  class ChildReport1 implements ReportInterface<typeof CHILD_QUERY1, 'a'> {
-    static output = ['a'];
-    static query = CHILD_QUERY1;
-    static key = 'Child1';
-    fetch() {
-      return {};
-    }
-
-    transform() {
-      return ['a', { a: 'a' }] as [string, { a: string }];
-    }
-  }
-  class ChildReport2 implements ReportInterface<typeof CHILD_QUERY2, 'b'> {
-    static output = ['b'];
-    static query = CHILD_QUERY2;
-    static key = 'Child2';
-    fetch() {
-      return {};
-    }
-
-    transform() {
-      return ['a', { b: 'b' }] as [string, { b: string }];
-    }
-  }
-  it('adjusts underscores to camel cases', () => {
+describe('Ads API Types', function () {
+  it('adjusts underscores to camel cases', function () {
     const test1: AssertEqual<CamelCase<'my_underscore'>, 'myUnderscore'> = true;
-    expect(test1).toBeDefined();
+    expect(test1).not.to.be.undefined;
   });
 
-  it('does not camel case where not necessary', () => {
+  it('does not camel case where not necessary', function () {
```

Comment:
TIL that mocha uses `this` https://mochajs.org/#arrow-functions.

No action required, just documenting.